packages/fx-core/src/common/deps-checker/util/cpUtils.ts

```diff
@@ -1,7 +1,7 @@
-/* eslint-disable @typescript-eslint/no-namespace */
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+/* eslint-disable @typescript-eslint/no-namespace */
```

Comment:
what's this?

Reply:
It is an existed eslint disable in line 1 for no warning for using namespace.
Just move it under the file header.